src/server/apis/dash-data.ts

```diff
@@ -189,8 +199,24 @@ export class DashData {
           }
         }
 
+        prIds.push(pr.id);
         incomingPrs.push(reviewedPr);
       }
+
+      // Incoming review requests.
+      for (const pr of viewerPrsResult.data.reviewRequests.nodes || []) {
+        if (!pr || pr.__typename !== 'PullRequest' || prIds.includes(pr.id)) {
```

Comment:
The naming here isn't clear `prIds.includes(pr.id)`, how do these ID's differ?

Reply:
Updated name to prsShown. I just stumbled upon a weird case where GitHub shows it in review requests and there is a review on it. So this prevents it from showing up twice.